CRM/Contact/Import/Parser/Contact.php

```diff
@@ -1704,4 +1710,42 @@ protected function checkForDuplicateExternalIdentifiers(string $externalIdentifi
     }
   }
 
+  /**
+   * @param string $countyID
+   * @param string|int|null $stateProvinceID
+   * @param string|int|null $countryID
+   *
+   * @return string|int
+   * @throws \CRM_Core_Exception
+   */
+  private function tryToResolveCounty(string $countyID, $stateProvinceID, $countryID) {
+    if (!isset(\Civi::$statics[$countyID . '_' . $stateProvinceID . '_' . $countyID])) {
```

Comment:
country should be county or vice-versa

Reply:
oh good point - will fix

Reply:
Does the english dictionary have a gitlab where I can propose changing the word county to something that is at least two letters different from country?